pkgs/development/web/deno/default.nix

```diff
@@ -40,8 +40,8 @@ rustPlatform.buildRustPackage rec {
     installShellFiles
   ];
   buildInputs = lib.optionals stdenv.isDarwin (
-    [ libiconv darwin.libobjc ] ++
-    (with darwin.apple_sdk_11_0.frameworks; [
+    [ libiconv darwin.libobjc ]
+    ++ (with darwin.apple_sdk_11_0.frameworks; [
```

Comment:
These changes are not required for the bump.

Reply:
Correct. In the PR description, I included:

> Formatted changed files with nixpkgs-fmt

I know it isn't in the document, but I was trying to keep things consistent. If this is a friction point, definitely let me know and I will avoid formatting files this way in the future.

Thank you for the review!

Reply:
Hey, thanks for getting back to me!

Just wanted to say I appreciated the effort to keep the codebase clean. I went ahead and merged the PR since those changes were minor anyway. For future updates, it'd be great if we could keep commits a bit more focused, scoped - one for the update itself and another for any formatting.

Keeps things super clear for everyone. Thanks again for the work!

Reply:
Thank you for the response! One clarification request: the same PR but as two commits is preferred, or two PRs with different commits?

Appreciate your time very much!